public/pages/EditFeatures/containers/EditFeatures.tsx

```diff
@@ -243,11 +247,72 @@ export function EditFeatures(props: EditFeaturesProps) {
     }
   };
 
+  const renderAdvancedSettingsToggle = () => (
+    <EuiText
+      className="content-panel-subTitle"
+      onClick={()=>{setShowAdvancedSettings(!showAdvancedSettings);}}
+    >
+      <EuiLink>
+        {showAdvancedSettings ? 'Hide' : 'Show'}
```

Comment:
For a new detector, it's ok to hide the window size setting to keep the UI simple and don't bring too much learning burden about the advanced setting. But if user choose to change the default shingle size and update the detector, why we still hide this window size ? For example, user create a detector, then go to this model configuration page, add one feature and set window size as 4, should we still hide the window size for this detector when user edit it later?

Reply:
I discussed this with the UX designer and we decided to keep this behavior. There isn't an easy way to tell if a user has explicitly set the window size on a detector in the past without storing this information in the ES detectors index, and it doesn't make much sense to modify the backend to store information that is purely a UX concern. Furthermore, the UX designer said the current behavior is consistent with advanced configuration panels in the AWS console.

Reply:
Thanks for confirming with UX designers. Should be easy if we show the advanced setting if its value is different with default value. 
Not a big impact to user experience for showing/hiding advanced setting if detector not using default setting. Respect UX designer's choice. Let's keep it as is.